### PR TITLE
Add annotation information when describing new-app results

### PR DESCRIPTION
**Pull request by smarterclayton**
Also reference that containers can access the service via DNS

### CODE REVIEW

**Comment by smarterclayton** [comment]
```
--> Found Docker image 72473b1 (24 hours old) from Docker Hub for "openshift/postgresql-92-centos7"

    PostgreSQL 9.2 
    -------------- 
    PostgreSQL is an advanced Object-Relational database management system

    Tags: database, postgresql, postgresql92

    * An image stream will be created as "db2:latest" that will track this image
    * This image will be deployed in deployment config "db2"
    * Port 5432/tcp will be load balanced by service "db2"
    * Other containers can access this service through the hostname "db2"

--> Creating resources with label app=db2 ...
    ImageStream "db2" created
    DeploymentConfig "db2" created
    Service "db2" created
--> Success
    Run 'oc status' to view your app.
```

File: pkg/generate/app/cmd/describe.go [hunk 1]
```diff
@@ -66,12 +66,52 @@ func describeLocatedImage(refInput *app.ComponentInput, baseNamespace string) st
 	}
 }
 
+func inputAnnotations(match *app.ComponentMatch) map[string]string {
+	if match == nil {
+		return nil
+	}
+	base := make(map[string]string)
+	if match.Image != nil {
```
Comment (csrwng) [inline pkg/generate/app/cmd/describe.go]: if image = match.Image; image != nil {

**Comment by csrwng** [comment]
other than minor nit, LGTM

**Comment by smarterclayton** [comment]
[merge]

**Pull request merged**